.travis.yml

```diff
@@ -32,10 +33,16 @@ addons:
   apt:
     packages:
       - swig
+  firefox: latest
```

Comment:
Can we add a comment that links to https://docs.travis-ci.com/user/firefox/ - I first thought that this might link to nightly or so and was about to complain but then realised that "latest" here is just the most recent official release of firefox, so it's stable.